core/src/main/java/org/apache/accumulo/core/client/rfile/RFile.java

```diff
@@ -172,17 +175,27 @@ public interface ScannerOptions {
      * {@link Property#TABLE_PREFIX} may be accepted and used. For example, cache and crypto
      * properties could be passed here.
      *
+     * <p>
+     * Starting with {@code 2.1.4}, {@link PluginEnvironment#getConfiguration(TableId)} (obtained by
+     * {@link IteratorEnvironment#getPluginEnv()}) will return the properties passed in here.
+     *
```

Comment:
I'm not sure these comments about "starting with 2.1.4" make sense to include here. It should have always worked this way. This is just fixing a bug. I don't think we need to document the implementation details of the corrected behavior for the bug fix in the javadoc.

Reply:
Okay, I will remove this in my next commit. Should I just remove the mention of `2.1.4` or that new part entirely?